_includes/navbar.html

```diff
@@ -1,79 +1,114 @@
 <header class="header">
-  <div id="navbar-original" hidden>
-    <nav class="navbar navbar-default">
-      <div class="container">
-        <div class="navbar-header">
-          <button type="button" class="navbar-toggle collapsed" data-toggle="collapse" data-target="#bs-example-navbar-collapse-1" aria-expanded="false">
-            <span class="sr-only">Toggle navigation</span>
-            <span class="icon-bar"></span>
-            <span class="icon-bar"></span>
-            <span class="icon-bar"></span>
-          </button>
-          <a class="navbar-brand" href="/"><strong>Gruntwork</strong>.io</a>
-        </div>
-        <div class="collapse navbar-collapse" id="bs-example-navbar-collapse-1">
-          <ul class="nav navbar-nav navbar-right">
-            <li class="js-nav-item-central"><a href="/how-it-works/" ga-on="click" ga-event-category="nav-{{ page.path | slugify }}" ga-event-action="how-it-works">How it Works</a></li>
-            <li class="dropdown js-nav-item-central">
-              <a href="#" class="dropdown-toggle" data-toggle="dropdown" role="button" aria-expanded="false">Products <i class="fa fa-caret-down" aria-hidden="true"></i></a>
-              <ul class="dropdown-menu dropdown-menu-left" role="menu">
-                <li><a href="/infrastructure-as-code-library/" ga-on="click" ga-event-category="nav-{{ page.path | slugify }}" ga-event-action="infrastructure-as-code-library">Infrastructure as Code Library</a></li>
-                <li><a href="/training/" ga-event-category="nav-{{ page.path | slugify }}" ga-event-action="training-library">DevOps Training Library</a></li>
-                <li><a href="/reference-architecture/" ga-event-category="nav-{{ page.path | slugify }}" ga-event-action="reference-architecture">Reference Architecture</a></li>
-                <li><a href="/houston/" ga-event-category="nav-{{ page.path | slugify }}" ga-event-action="reference-architecture">Gruntwork Houston</a></li>
+    {% if page.url contains "guides" %}
```

Comment:
As I mentioned in the top-level comment, I don't think anything needs to change in the nav yet. We are still working on that.

Reply:
Okay. WIll revert this too